SECP-service/src/main/java/com/visucius/secp/models/Filter.java

```diff
@@ -0,0 +1,111 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "Filters")
+@NamedQueries(
+    {
+        @NamedQuery(
+            name = "com.visucius.secp.models.Filter.findByName",
+            query = "from Filter f where f.name = :name"
+        ),
+        @NamedQuery(
+            name = "com.visucius.secp.models.Filter.findAllFilters",
```

Comment:
you don't need this query. use the findall method in your dao

Reply:
The find all method in the DAO is making use of this query.